weave/index/index_sql.py

```diff
@@ -274,7 +274,8 @@ def to_pandas_df(self, max_rows=1000, offset=0, **kwargs):
         Parameters
         ----------
         max_rows: int (default=1000)
```

Comment:
max_rows: int or None (default=1000)

Reply:
All occurrences of this have been updated. Thanks for catching this.